fearless/Common/Protocols/ErrorStateViewProtocol.swift

```diff
@@ -0,0 +1,40 @@
+import UIKit
+
+protocol ErrorStateViewProtocol: AnyObject {
```

Comment:
I think we can just return ErrorView from ```viewForEmptyState``` delegate method of EmptyStateView. And also we can set ViewController as delegate for ErrorView and call presenter ```reload``` function in delegate method. I think It will be more clear.

Currently, imagine a situation when someone comes to fix something in view. And it will be hard to understand that this tap on error view is actually handled not by the controller but by closure in the bottom of the presenter :)